### PR TITLE
[BACK-2679] Resolve dependabot issues

### PR DESCRIPTION
**Pull request by darinkrauss**
- Resolve dependabot issues
- Update to node v20.11.0
- Remove request dependency
- Resolve lint issues
- Add lint via jshint
- Add .nvmrc
- Add .travis.yml
- Bump version
- https://tidepool.atlassian.net/browse/BACK-2679

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v20.11.0
```
Comment (jh-bate) [inline .nvmrc:1]: should be `v20.10.0`?

**Review by jh-bate — APPROVED** [review]
one note on version

File: .nvmrc (hunk 1 above)
Comment (darinkrauss) [reply .nvmrc:1]: Updated PR description.

**Review by jh-bate — APPROVED** [review]
one question re alpha tag - otherwise LGTM

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tidepool-seagull-client",
-  "version": "0.1.9",
+  "version": "0.1.11-alpha.6",
```
[the remark this answers is not in the document]
Comment (darinkrauss) [reply package.json]: Yep, fixed.

**Review by jh-bate — APPROVED** [review]
LGTM

**Pull request merged**